### PR TITLE
simplistic article view

### PR DESCRIPTION
**Pull request by ldalves**
make simplistic article work... let's make the improvements on another PR.

### CODE REVIEW

File: src/App/Article/index.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import Link from 'App/shared/components/Link';
+
+const ContainerContent = styled(Container)`
+  &&& {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+`;
+
+const nameFormatter = ({ first, middle, last }) => {
```
Comment (freva) [inline src/App/Article/index.js]: Need to extract this to shared utils

File: src/App/Article/index.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  return (matches ? matches.join('') + ' ' : '') + last;
+};
+
+function Authors({ authors }) {
+  const limit = authors.length;
+  return (
+    <Header.Subheader>
+      {authors
+        .map(nameFormatter)
+        .slice(0, limit)
```
Comment (freva) [inline src/App/Article/index.js]: Redundant

File: src/App/Article/index.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      <React.Fragment>
+        {response.root.children.map((article, i) => (
+          <ResultCard key={i} {...article} />
+        ))}
+      </React.Fragment>
+    </Tab.Pane>
+  );
+}
+
+function CitedBy({ citedBy }) {
```
Comment (freva) [inline src/App/Article/index.js]: This crashes if citedBy does not exist

**Comment by bratseth** [comment]
Great!

File: src/App/Article/index.js [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  return citedBy.map((id, _) => {
```
Comment (freva) [inline src/App/Article/index.js]: Redundant index

Comment (freva) [reply src/App/Article/index.js]: This also needs to be limited/paginated

File: src/App/Article/index.js [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    `/document/v1/covid-19/doc/docid/${id}`
+  ).state();
+
+  if (loading) return <Loading message="Loading..." />;
+  if (error)
+    return <Error message={error.message || `Failed to load article #${id}`} />;
+
+  console.log(response);
+
+  const panes = [
```
Comment (freva) [inline src/App/Article/index.js]: Consider making Abstract a panel

File: src/App/Article/index.js [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    {
+      menuItem: 'Related',
+      render: () => <Related title={response.fields.title} />,
+    },
+    {
+      menuItem: 'Cited by',
```
Comment (freva) [inline src/App/Article/index.js]: Consider adding floating icon (https://react.semantic-ui.com/elements/label/#types-floating) for # of citations in tab title

File: src/App/Article/index.js (hunk 1 above)
Comment (ldalves) [reply src/App/Article/index.js]: 👍

File: src/App/Article/index.js (hunk 2 above)
Comment (ldalves) [reply src/App/Article/index.js]: 👍

File: src/App/Article/index.js (hunk 3 above)
Comment (ldalves) [reply src/App/Article/index.js]: @jonmv I guess you have fixed this one?

File: src/App/Article/index.js (hunk 4 above)
Comment (ldalves) [reply src/App/Article/index.js]: @jonmv

**Pull request merged**